### PR TITLE
Ibrahimahmed443/sol 1611

### PR DESCRIPTION
**Pull request by ibrahimahmed443**

### CODE REVIEW

**Comment by ibrahimahmed443** [comment]
work in progress @afzaledx @douglashall

File: edx_proctoring/backends/software_secure.py [hunk 1]
```diff
@@ -392,6 +394,20 @@ def _get_payload(self, exam, context):
...
+                "examEndDate": end_time_str,
+                "noOfStudents": 1,
+                "examID": exam['id'],
+                "courseID": exam['course_id'],
+                "firstName": first_name,
+                "lastName": last_name
+            }
+        }
+        if self.send_email_to_ss:
+            org_extra['orgExtra']["email"] = context['email']
```
Comment (douglashall) [inline edx_proctoring/backends/software_secure.py]: This should be:
```
org_extra = {
    "examStartDate": start_time_str,
    "examEndDate": end_time_str,
    "noOfStudents": 1,
    "examID": exam['id'],
    "courseID": exam['course_id'],
    "firstName": first_name,
    "lastName": last_name
}
if self.send_email_to_ss:
    org_extra["email"] = context['email']
```

File: edx_proctoring/backends/software_secure.py [hunk 2]
```diff
@@ -47,7 +47,8 @@ class SoftwareSecureBackendProvider(ProctoringBackendProvider):
     """
 
     def __init__(self, organization, exam_sponsor, exam_register_endpoint,
-                 secret_key_id, secret_key, crypto_key, software_download_url):
+                 secret_key_id, secret_key, crypto_key, software_download_url,
+                 send_email_to_ss=False):
```
Comment (douglashall) [inline edx_proctoring/backends/software_secure.py]: Let's rename this to just `send_email`.

**Comment by douglashall** [comment]
:+1:  When final comment is addressed and commits are squashed.

**Pull request merged**